src/components/List/ListItem/ListTarget.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { DropTarget } from 'react-dnd';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { settings } from '../../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+export const TargetSize = {
+  Third: 'third',
+  Half: 'half',
+  Full: 'full',
+};
+
+const ListTargetPropTypes = {
+  connectDropTarget: PropTypes.func.isRequired,
+  isOver: PropTypes.bool,
+  targetPosition: PropTypes.string.isRequired,
+  targetSize: PropTypes.oneOf([TargetSize.Third, TargetSize.Half, TargetSize.Full]),
+  // eslint-disable-next-line react/no-unused-prop-types
+  targetOverride: PropTypes.string,
+};
+
+const ListTargetDefaultProps = {
+  isOver: false,
+  targetSize: TargetSize.Third,
+  targetOverride: null,
+};
+
+const ListTarget = ({ connectDropTarget, targetPosition, targetSize, isOver }) => {
+  let height = 33; // defaults to TargetSize.Third
+
+  if (targetSize === TargetSize.Full) {
+    height = 100;
+  } else if (targetSize === TargetSize.Half) {
+    height = 50;
+  }
+
+  return (
+    <div
+      data-testid="list-target"
+      className={classnames(`${iotPrefix}--list-item-editable--drop-target-${targetPosition}`, {
+        [`${iotPrefix}--list-item-editable--drop-target-${targetPosition}__over`]: isOver,
+      })}
+      style={{
+        height: `${height}%`,
+      }}
+      ref={instance => {
+        if (connectDropTarget) {
+          connectDropTarget(instance);
+        }
+      }}
+    />
+  );
+};
+
+/* istanbul ignore next */
```

Comment:
I don't think this should be ignored. Is it dropping you below the threshold?

Reply:
Yes. we should remove these.